src/entities/partner-feature.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Column, Entity, JoinTable, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { BaseEntity } from '../entities/base.entity';
+import { FeatureEntity } from './feature.entity';
+import { PartnerEntity } from './partner.entity';
+
+@Entity({ name: 'partner_feature' })
+export class PartnerFeatureEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid', { name: 'partnerFeatureId' })
+  id: string;
+  @Column()
+  active: boolean;
+
+  @Column()
+  partnerId: string;
+  @ManyToOne(() => PartnerEntity, (partnerEntity) => partnerEntity.partnerFeature)
+  @JoinTable({ name: 'partner', joinColumn: { name: 'partnerId' } })
+  partner: PartnerEntity;
+
+  @Column()
+  featureId: string;
+  @ManyToOne(() => FeatureEntity, (featureEntity) => featureEntity.partnerFeature, {
```

Comment:
I might be wrong about this - but I thought the onDelete would go on the feature entity. 

Sort of similar to how the onDelete is on the partner entity for the partnerAccess relationship. I'm not a typeorm expert so I might be wrong on this

Reply:
I think that is wrong - I watched this youtube https://www.youtube.com/watch?v=S1pxdAL2Dz8 and it seems to imply that the oncascade should be on the join table. I actually didn't put the cascade on the partner so I will do that